src/esp/assets/ResourceManager.cpp

```diff
@@ -1790,6 +1790,10 @@ bool ResourceManager::instantiateAssetsOnDemand(
   ObjectAttributes::ptr ObjectAttributes =
       getObjectAttributesManager()->getObjectByHandle(objectTemplateHandle);
 
+  if (!ObjectAttributes) {
```

Comment:
Can we add an appropriate error the logger perhaps so that users know what went wrong?

Reply:
The error-logging here is already quite verbose:
```
E0412 08:38:20.836441 2529307 ManagedContainerBase.h:288] ManagedContainer::getObjectByHandle : Unknown Object managed object handle :invalid_handle. Aborting
E0412 08:38:20.836445 2529307 PhysicsManager.cpp:83] PhysicsManager::addObject : ResourceManager::instantiateAssetsOnDemand unsuccessful. Aborting.
```